scst-store/amr/graphql-query.hbs.md

```diff
@@ -0,0 +1,261 @@
+# AMR Graphql Querying
+
+This page contains information on how to connect to the GraphQL playground, and how to query each supported data model.
+See [AMR Data Models](data-model-and-concepts.hbs.md) for more information on the AMR data models.
+
+## Connecting to Graphql Playground
+
+The recommended configuration is to enable ingress. The Supply Chain Security Tools for Tanzu – Store and Artifact Metadata Repository (AMR) packages share the same ingress configuration. 
+As such setting `ingress_domain` for the Store will apply the same value for the AMR.
+To connect to the AMR GraphQL playground when ingress is enabled, go to `https://artifact-metadata-repository.<ingress-domain>/play`.
+On this page users will be able to write and execute their own GraphQL queries to fetch data from the AMR.
```

Comment:
small suggestion: `will` ->  `are` since this isn't a future feature and it's something that they can do now provided they are set up.